solutions/standard/variables.tf

```diff
@@ -354,3 +354,15 @@ variable "elasticsearch_full_version" {
   type        = string
   default     = null
 }
+
+variable "image_port" {
+  description = "The port which is used to connect to the port that is exposed by the container image."
+  type        = number
+  default     = 5601
+}
+
+variable "managed_domain_mappings" {
```

Comment:
again I would use `kibana` in the variable name

Reply:
Done